packages/gasket-cli/package.json

```diff
@@ -76,14 +76,12 @@
     "inquirer": "6.3.1",
     "json5": "^2.1.0",
     "lodash.defaultsdeep": "^4.6.1",
-    "map-stream": "^0.1.0",
     "mkdirp": "^1.0.4",
     "ora": "^3.4.0",
     "pump": "^3.0.0",
     "semver": "^7.3.4",
     "tar-fs": "^2.0.0",
-    "untildify": "^4.0.0",
-    "vinyl-fs": "^3.0.3"
```

Comment:
👍 this also drops one of the vulnerability `glob-parent@3.1.0`
```
│ └─┬ vinyl-fs@3.0.3
│   └─┬ glob-stream@6.1.0
│     └── glob-parent@3.1.0
```

Reply:
👍 Exactly - the main reason instigating this change.